styles/all/template/event/forumlist_body_category_header_before.html

```diff
@@ -0,0 +1,7 @@
+<!-- IF forumrow.S_IS_CAT -->
+<a class="category <!-- IF forumrow.S_FORUM_HIDDEN -->hidden-category<!-- ENDIF -->"
+	id="category{forumrow.FORUM_ID}"
+	data-id="{forumrow.FORUM_ID}"
```

Comment:
we should make this `fid_{forumrow.FORUM_ID}` if we want to use fid_1, fid_2 etc for our ids